integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -848,12 +850,15 @@ public void testCreateNewCluster() {
    * To: "image: container-registry.oracle.com/middleware/weblogic:14.1.1.0-11"
    * Verify all the pods are restarted and back to ready state
    * Verify the admin server is accessible and cluster members are healthy
+   * This test will be aborted if the image tag is: 14.1.1.0-11
    */
   @Order(5)
   @Test
   @DisplayName("Verify server pods are restarted by updating image name")
   public void testUpdateImageName() {
-
+    logger.info("In the testUpdateImageName() the image version is: " + WEBLOGIC_IMAGE_TAG);
+    logger.info("This test will be ABORTED if WebLogic image version is:  14.1.1.0-11");
```

Comment:
Define a WEBLOGIC_IMAGE_LATEST constant and use it here. That way when a new image comes we can always abort this test when it is latest.